db/fixtures/miq_product_features.yml

```diff
@@ -5172,45 +5172,45 @@
         :feature_type: admin
         :identifier: embedded_configuration_script_source_delete
       - :name: Refresh Repositories
-        :description: Ansible Repositories Refresh
+        :description: Automation Manager Repositories Refresh
         :feature_type: control
         :identifier: embedded_configuration_script_source_refresh
       - :name: Edit Tags
-        :description: Edit Tags for the selected Ansible Repositories
+        :description: Edit Tags for the selected Automation Manager Repositories
         :feature_type: control
-        :identifier: ansible_repository_tag
-  - :name: Playbooks
-    :description: Everything under Playbooks
+        :identifier: embedded_configuration_script_source_tag
+  - :name: Configuration Scripts
+    :description: Everything under Configuration Scripts
     :feature_type: node
     :identifier: embedded_configuration_script_payload
     :children:
     - :name: View
-      :description: View Playbooks
+      :description: View Configuration Scripts
```

Comment:
Should this be `View Configuration Script Payloads` ?  The identifier is `embedded_configuration_script_payload_view` and https://github.com/ManageIQ/manageiq/pull/22971/files#diff-cdcd147fb85ef58d008a4ad0cfa3ddfea649382cb062407e2342aff07362b6cfR5208 has `Refresh Configuration Script Payloads`

Reply:
I'll change them all to "Payloads"

Reply:
Updated